examples/bert/bert_auto_opt_gpu copy.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
typo: remove `copy` from the file name.

Reply:
Thanks~ I just  send this PR to clarify bare transformer olive config user need to give.  There are separated PRs to achieve this `bare` json.
Please refer to description of this PR.

Reply:
Will close this once the separated PRs are done.